opus/application/test_api/api_test_helper.py

```diff
@@ -166,3 +168,29 @@ def _run_csv_equal_file(self, url, exp_file):
         print('Expected:')
         print(expected)
         self.assertEqual(resp, expected)
+
+    def _run_zip_equal(self, url, expected, response_type='json'):
+        print(url)
+        response = self._get_response(url)
+        self.assertEqual(response.status_code, 200)
+        if response_type == 'json':
+            jdata = json.loads(response.content)
+            file = jdata['filename']
+            filename = file.replace(settings.TAR_FILE_URL_PATH, '')
+            path = settings.TAR_FILE_PATH + filename
```

Comment:
Can't you just say file.replace(settings.TAR_FILE_URL_PATH, settings.TAR_FILE_PATH)?